ledger/sandbox-on-x/reference.conf

```
@@ -160,13 +188,17 @@ ledger {
         }
 
         # The timeout for the synchornous interaction with package and party management services.
+        # TODO config rationalization: split into 'party-management-service-timeout' and
+        # 'package-management-service-timeout'
+        # TODO as above, Canton has "unified" the timeouts, so more like by type of timeout rather
+        #   by component
         management-service-timeout = 2 minutes
 
         # Max inbound message size in bytes.
         # You can use this parameter to increase (or decrease) the maximum size of a
         # GRPC message. Often, DARs or transactions can become larger than the default
         # of 4194304 bytes (4 MB). Increasing this will allow for larger transactions,
-        # at the expense of processing time.
+        # at the expense of processing time.        
         max-inbound-message-size = 67108864 #64 * 1024 * 1024
```

```suggestion
        max-inbound-message-size = 9.5 * 1024 * 1024
```
as proposed in [2021-10 - Design: Explicit Daml-LF and Ledger API Limits](https://docs.google.com/document/d/1kjiHIQfTgsIMynpjqPAjRBOS6bgoPpWxpC1iKfpEeYA/edit#bookmark=id.mdievaanx2gj)

---

Will dar uploads from spider-app fit into that limit?

---

No, they will have to change the default.

I would like the defaults to give guidance to users about what they should aim for.

The Spider .dar size is a problem waiting to blow up ...